cabal-install/Distribution/Client/ProjectPlanning.hs

```diff
@@ -1859,14 +1861,23 @@ cabalPkgname = PackageName "Cabal"
 basePkgname  = PackageName "base"
 
 
-legacyCustomSetupPkgs :: Platform -> [PackageName]
-legacyCustomSetupPkgs (Platform _ os) =
+legacyCustomSetupPkgs :: Compiler -> Platform -> [PackageName]
+legacyCustomSetupPkgs compiler (Platform _ os) =
     map PackageName $
         [ "array", "base", "binary", "bytestring", "containers"
-        , "deepseq", "directory", "filepath", "pretty"
-        , "process", "time" ]
+        , "deepseq", "directory", "filepath", "old-time", "pretty"
+        , "process", "time", "transformers" ]
+     --TODO: [research required] does old-time need to be conditional?
```

Comment:
I don't think so, this will just break old scripts that relied on `System.Time` being available by default.